int32 botId
int32 instruction
int32 forwardSpeed
int32 rotationalSpeed
